addon/_locales/zh_TW/messages.json

```diff
@@ -0,0 +1,196 @@
+{
+  "extensionName": {
+    "message": "OSM 智慧選單"
+  },
+  "extensionShortName": {
+    "message": "OSMenu"
```

Comment:
This message might be used by browsers when they need a short name.
Since `OSM 智慧選單` is already short, you can repeat it here if you think it makes things easier to understand